bazel/grpc_deps_1_15_1.patch

```diff
@@ -0,0 +1,155 @@
+diff --git a/bazel/grpc_deps.bzl b/bazel/grpc_deps.bzl
```

Comment:
I don't think you wanted to add this file?

Reply:
It's autogenerated by `git diff`, I think this is needed so the tool knows which file to patch. The `patches` flag does not allow you to specify the file path.